### PR TITLE
Finished TEST: Complete order by adding payment type #14

### LINKED ISSUE #14: TEST: Complete order by adding payment type
Add an integration test to `tests/order.py` that tests adding a payment type to an order.

1. Create a payment type in setup
1. Perform PUT to an order with the payment type id in the request body
1. Perform GET to the order to verify the payment type is assigned

### PR DESCRIPTION
**Pull request by Kamiran79**
Description of PR that completes issue here...

## Changes

- Item 1: line 110 on views/product.py there was a bug noticed when try apply test because the image page and I was clean fields so I had exclude that field
- Item 2: line 87-146 on tests/order.py adding two function on for create the payment type the second to put the payment to an order.

## Requests / Responses

If this PR contains code that defines a new request/response, or changes an existing one, please put the JSON representations here.

**Request**
```json
{
}
```

**Response**

Ran 3 tests in 0.307s OK

```json
{
}
```

## Testing

Description of how to test code...

- [ ] Run python manage.py test tests.order -v 1

## Related Issues

- Fixes #14

### CODE REVIEW

File: tests/order.py [hunk 1]
```diff
@@ -27,6 +29,7 @@ def setUp(self) -> None:
         data = { "name": "Kite", "price": 14.99, "quantity": 60, "description": "It flies high", "category_id": 1, "location": "Pittsburgh" }
         self.client.credentials(HTTP_AUTHORIZATION='Token ' + self.token)
         response = self.client.post(url, data, format='json')
+        #import pdb; pdb.set_trace()
```
Comment (hannahhall) [inline tests/order.py]: Remove this comment

File: tests/order.py [hunk 2]
```diff
@@ -81,4 +84,63 @@ def test_remove_product_from_order(self):
...
+        self.test_create_paymenttype()
+
+        url = "/cart"
+        data = { "product_id": 1 }
+        self.client.credentials(HTTP_AUTHORIZATION='Token ' + self.token)
+        response = self.client.post(url, data, format='json')
+
+
+
+        # DEFINE NEW PROPERTIES FOR GAME
```
Comment (hannahhall) [inline tests/order.py]: And this one

File: tests/order.py [hunk 3]
```diff
@@ -81,4 +84,63 @@ def test_remove_product_from_order(self):
...
+
+        self.client.credentials(HTTP_AUTHORIZATION='Token ' + self.token)
+        payment_data = {
+            "payment_type": 1
+        }
+        response = self.client.put(f"/profile/cart/", payment_data, format="json")
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+
+        # GET GAME AGAIN TO VERIFY CHANGES
```
Comment (hannahhall) [inline tests/order.py]: and this one

**Review by hannahhall — APPROVED** [review]
Looks good just need to clean up comments

File: tests/order.py (hunk 1 above)
Comment (Kamiran79) [reply tests/order.py]: Can I leave that line just for reminder ?

Comment (Kamiran79) [reply tests/order.py]: Updated

**Pull request merged**